australian-guide-to-legal-citation.csl

```diff
@@ -199,11 +196,10 @@
         <text variable="title" font-style="italic" text-case="title" form="short"/>
       </if>
       <else>
-        <text variable="title" quotes="true" text-case="title" form="short"/>
+        <text variable="title" form="short" text-case="title" quotes="false"/>
```

Comment:
Yep. You're right. I broke it. Sorry. Very experienced with AGLC, not so experienced with coding. I think I've got it now, with comments, but I'll run a few documents through it tomorrow to check i haven't broken anything else before I resubmit.